docs-chef-io/content/workstation/reusable/md/workstation_supported_platforms.md

```diff
@@ -1,8 +1,8 @@
-| Platform                          | Architecture    | Version                                                                    |
-|-----------------------------------| ----------------| ---------------------------------------------------------------------------|
-| Amazon Linux                      | x86_64          | 2.x                                                                        |
-| macOS                             | x86_64, arch64  | 11.x, 12.x                                                                 |
-| Debian                            | x86_64          | 10, 11                                                                     |
-| Red Hat Enterprise Linux / CentOS | x86_64          | 7.x, 8.x, 9.x                                                              |
-| Ubuntu                            | x86_64          | 18.04, 20.04, 22.04                                                        |
-| Windows                           | x64             | 10, 11, Server 2012, Server 2012 R2, Server 2016, Server 2019, Server 2022 |
+| Platform                          | Architecture    | Version                                                                       |
+|-----------------------------------| ----------------| ------------------------------------------------------------------------------|
+| Amazon Linux                      | x86_64, arch64  | 2.x, 2023                                                                     |
+| macOS                             | x86_64, arch64  | 11.x, 12.x                                                                    |
+| Debian                            | x86_64          | 10.x, 11.x                                                                    |
```

Comment:
Dibian 10 extended support is also almost done and only 11.x and 12.x will be supported by debian. https://endoflife.date/debian